### PR TITLE
feat: IT’S REALLY RUDE DOXXING PPL LYN

### PR DESCRIPTION
**Pull request by mavi0**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import discord
 from discord.ext import tasks
 
+name = "notAlyx"
 client = discord.Client(intents=discord.Intents.default())
-message = "Is it your birthday today, Alyx?"
+message = "Is it your birthday today, " + name + "?"
```
Comment (WillFantom) [inline bot.py:6]: IMG_5689

Comment (mavi0) [reply bot.py:6]: IMG_1130

**Comment by Lynverted** [comment]
hippo

**Comment by mavi0** [comment]
DISAPPOINT

**Comment by WillFantom** [comment]
Open source is DED 
sad

**Pull request closed without merging**